pages/about.tsx

```diff
@@ -29,9 +31,19 @@ const AboutPage: React.FC = () => {
       <AboutHero isMobile={isMobile} image={HeroImage.src} />
       <AboutGetInvolved isMobile={isMobile} />
       {isMobile ? null : <AboutMembership />}
-      <AboutBoard isMobile={isMobile} />
+      <AboutBoard isMobile={isMobile} board={board} />
     </>
   );
 };
 
 export default AboutPage;
+
+export async function getStaticProps() {
+  const boardData = await getBoardData();
+  return {
+    props: {
+      board: boardData || [],
+    },
+    revalidate: 1 * 60 * 60 * 24, // once every day (in seconds)
```

Comment:
question - is there a way to manually trigger a revalidate / if we add something to the board spreadsheet, is there a way to trigger an instantaneous change?